{
  "pack_xdsl_access_diagnostic_title": "Diagnostic de votre ligne",
  "pack_xdsl_access_diagnostic_wizard_title": "Assistant de résolution de problèmes",
  "pack_xdsl_access_diagnostic_launch": "Lancer un diagnostic",
  "pack_xdsl_access_diagnostic_assistant_launch": "Lancer l'assistant",
  "pack_xdsl_access_diagnostic_remaining": "{{value}} restant(s)",
  "pack_xdsl_access_diagnostic_wizard_radio_probleme1": "Le diagnostic de ligne indique un problème de synchronisation.",
  "pack_xdsl_access_diagnostic_wizard_radio_probleme2": "J'ai une perte de connexion totale.",
  "pack_xdsl_access_diagnostic_wizard_radio_probleme3": "Mon opérateur détecte une connexion mais je ne peux pas me connecter.",
  "pack_xdsl_access_diagnostic_general": "Rapport de ligne",
  "pack_xdsl_access_diagnostic_time": "Heure du diagnostic",
  "pack_xdsl_access_diagnostic_access_incident_ongoing": "Un incident OVHcloud est en cours de résolution",
  "pack_xdsl_access_diagnostic_access_works": "Aucun incident",
  "pack_xdsl_access_diagnostic_access_fails": "Incident",
  "pack_xdsl_access_diagnostic_modem_connected": "Statut de connexion",
  "pack_xdsl_access_diagnostic_modem_state_disconnected": "Votre équipement OVHcloud est déconnecté",
  "pack_xdsl_access_diagnostic_modem_state_connected": "Votre équipement OVHcloud est connecté",
  "pack_xdsl_access_diagnostic_modem_state_running": "Équipement en cours de test…",
  "pack_xdsl_access_diagnostic_active_on_lns": "Passerelle de sortie LNS",
  "pack_xdsl_access_diagnostic_lns_active_ok": "Votre équipement est actif",
  "pack_xdsl_access_diagnostic_lns_active_ko": "Votre équipement n'est pas actif",
  "pack_xdsl_access_diagnostic_ping": "Réponse aux pings",
  "pack_xdsl_access_diagnostic_ping_ko": "Votre équipement ne répond pas aux pings",
  "pack_xdsl_access_diagnostic_ping_ok": "Votre équipement répond correctement aux pings",
  "pack_xdsl_access_diagnostic_ping_running": "Ping en cours…",
  "pack_xdsl_access_diagnostic_line": "Ligne n°{{name}}",
  "pack_xdsl_access_diagnostic_line_sync": "Synchronisation de la ligne",
  "pack_xdsl_access_diagnostic_line_sync_ko": "La ligne n'est pas synchronisée",
  "pack_xdsl_access_diagnostic_line_sync_ok": "La ligne est synchronisée",
  "pack_xdsl_access_diagnostic_line_proposed_profile_id": "Profil de connexion proposé",
  "pack_xdsl_access_diagnostic_line_current_profile_id": "Profil de connexion actuel",
  "pack_xdsl_access_diagnostic_line_line_test": "Test de la ligne",
  "pack_xdsl_access_diagnostic_line_line_test_actionPending": "En cours",
  "pack_xdsl_access_diagnostic_line_line_test_customerSideProblem": "Problème côté client",
  "pack_xdsl_access_diagnostic_line_line_test_error": "Erreur",
  "pack_xdsl_access_diagnostic_line_line_test_noProblem": "Fonctionnement OK",
  "pack_xdsl_access_diagnostic_line_line_test_ovhSideProblem": "Problème côté OVHcloud",
  "pack_xdsl_access_diagnostic_line_error_diag_launch": "Impossible de lancer le diagnostic.",
  "pack_xdsl_access_diagnostic_line_error_diag_poll": "Une erreur est survenue lors du diagnostic."
}
